runner/ansible/roles/post-metadata/tasks/post.yml

```diff
@@ -0,0 +1,10 @@
+- name: Post metadata to ARA
+  ara_record:
+    key: trento-metadata
+    value: '{{ metadata }}'
+    type: json
+  register: record
+
+- name: Print metadata
+  debug:
+    msg: '{{ record.value }}'
```

Comment:
I would add:
verbosity: 1
here, so the debug message is only done when a "-v" or more is provided.

Reply:
Thanks @brett060102,
Yes, it is a nice touch